cmd/thanos/compact.go

```diff
@@ -100,9 +100,9 @@ func registerCompact(m map[string]setupFunc, app *kingpin.Application) {
 		Hidden().Default("false").Bool()
 
 	// TODO(bplotka): Remove this flag once https://github.com/thanos-io/thanos/issues/297 is fixed.
```

Comment:
Does it still make sense for us to have this comment?

Reply:
I think it is better to remove this comment.